.github/workflows/issue-pr-triage.yml

```diff
@@ -26,7 +29,7 @@ jobs:
     name: Triage to Localization Project
```

Comment:
@LizBaker mentioned this might not work with the change to the github_token since the localization board does not live under the docs-website repository.

Reply:
update: definitely doesn't work.

Reply:
Yeah I thought that may be problem. We can kill that project board that's linked there and put it under the project repo.

Reply:
I submitted a change to fix this and setup a new project board under this repo. I'll let julien know